AI/OpenAI/README.md

```diff
@@ -0,0 +1,55 @@
+# OpenAI
+
+In this document, you will find how to install and run a simple OpenAI example.
+
+## What is OpenAI?
+
+See OpenAI [website](https://openai.com/) and
+[documentation](https://platform.openai.com/docs/introduction).
+
+## Security
+
+Please, ensure that you carefuly read [Harvard's guidelines](https://huit.harvard.edu/ai/guidelines) on AI.
+
+## Installation
+
+You can install OpenAI in a conda/mamba environment:
+
+```bash
+[jharvard@boslogin01 ~]$ salloc --partition test --time 01:00:00 --mem-per-cpu 4G -c 2
+[jharvard@holy8a24301 ~]$ module load python/3.10.12-fasrc01
+[jharvard@holy8a24301 ~]$ export PYTHONNOUSERSITE=yes
+[jharvard@holy8a24301 ~]$ mamba create -n openai_env openai
+```
+
+## Run OpenAI
+
+You will need to provide an OpenAI key. You can generate one from
+[https://platform.openai.com/api-keys](https://platform.openai.com/api-keys).
```

Comment:
Cannon can only deal with data level 2 and below. Data level 3 is only for FASSE. I am working with Warren to better understand what can/cannot be done.

Reply:
Checking with Ingrid on this.  This might be a "you need Harvard to create an account" or "you need Harvard to take your OpenAI account under their umbrella".

Reply:
Yep!  Some notes from Ingrid:

> Correct. The only way folks can use openAI and other genAI on their own is if it is not Harvard work related and/or public, non-sensitive data. Otherwise, they need to work with the school to discuss their needs and be sure they get setup with the appropriate contractual coverage and environment – esp. to avoid having the model learn from their input and leak sensitive information.

She mentioned [this](https://bokcenter.harvard.edu/artificial-intelligence) from the Bok Center, and some Harvard contacts (not sure if this comment is public so will ping in Slack)